### DISCUSSION TITLE
New in the game

### DISCUSSION THREAD

**Discussion by gotin23** [codewars/codewars.com#2762] [General]
Hello,
One month ago i start to learn javascript , i would like to know if codewars is good way to learn ?
And  witch level of kyu  i can reach for know if im good or not ?
(sorry for my poor english)

**Comment by savvy-itch** [comment]
Hi. My situation is exactly the same and I've been trying to solve some simple challenges for some time now. From my experience I can say it definitely helped me feel more confident in coding. I don't use it as a sole source for learning though - I work my way through a curriculums on other sources with a traditional theory/practice approach. Then I go here and try to apply my knowledge here and sometimes I even succeed)
If you'd like to team up in some competitive/supportive kind of way, I'd be up to it.

**Reply by gotin23** [reply]
Hey,
Sure this can be nice, you can add me on codewars and github if you want  :)

**Reply by savvy-itch** [reply]
Actually I can't)) because it's a real pain to comunicate/find someone on github and codewars. Could you message me by writing something in my rep? Or if not, you can drop some links/email to your social media here.

**Reply by gotin23** [reply]
yes you can find me on twitter if you want (@Gotin23)

**Reply by savvy-itch** [reply]
Ok, I've found you. Check it, I can't DM you cause you turned off DMs from strangers apparently

**Comment by moocstudent** [comment]
Welcoming!

**Reply by gotin23** [reply]
thanx you :)
